ci/pipelines/deploy-to-test.yml

```diff
@@ -1526,7 +1526,8 @@ jobs:
         passed: [build-and-push-telegraf-to-test-ecr]
       - get: nginx-proxy-ecr-registry-test
         trigger: true
-      - get: adot-ecr-registry-test
+      - get: adot-candidate
+        passed: [deploy-toolbox]
```

Comment:
This job is the deploy-toolbox job, you want to reference the previous job in the chain that adot-candidate would have gone through here, which would be `build-and-push-adot-candidate`